### ISSUE TITLE
\0 and \1 do not appear to work as booleans

### ISSUE THREAD

**Issue by oschwald** [sysread/TOML-Tiny#11]
The documentation suggests that `\0` and `\1` will work as booleans. However, this:

```
to_toml({ test => \1})
```

results in:

```
Can't use string ("SCALAR") as a SCALAR ref while "strict refs" in use at TOML/Tiny/Writer.pm line 98.
```

Glancing at the code, it appears that the topic variable is not the expected value

**Comment by sysread** [comment]
Fixed in `642d702`. I'll push that to cpan shortly.

**Comment by oschwald** [comment]
Thanks!
